lib/supabase/getAccessToken.js

```
@@ -0,0 +1,30 @@
+import getAccessTokenByRefreshToken from "./getAccessTokenByRefreshToken.js";
+import getAdminPresave from "./getAdminPresave.js";
+import supabase from "./serverClient.js";
+
+const getAccessToken = async () => {
+  try {
+    const presave = await getAdminPresave();
+    if (presave?.error) return { error };
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)